bloggies_frontend/src/App.tsx

```diff
@@ -9,9 +9,11 @@ import { getUserFavoritesFromAPI, getUserInfoFromAPI } from './redux/actionCreat
 import NavBar from './NavBar';
 import { CustomReduxState } from './custom';
 import { getCookie } from './helpers';
+import { ThemeProvider } from 'styled-components';
+import { defaultAppTheme, GlobalStyles, premiumAppTheme } from './theme';
 
 function App() {
-  const userId = useSelector((st: CustomReduxState) => st.user.id);
+  const currentUser = useSelector((st: CustomReduxState) => st.user);
```

Comment:
I'm curious was there a reason you changed this from userId to currentUser?

Reply:
Yea! The reason I changed it was because I wanted to use the redux state of the user's `membership_status` to figure out whether to change the UI theme or not. Before, I was only needed the userId to do some logic but now that component is using both the `user id` and `user membership_status` so I decided to select the whole user state and rename it as `currentUser` :)

Reply:
Thanks for the clarification that makes sense.